packages/components/app/styles/components/pagination.scss

```diff
@@ -49,6 +49,7 @@
 .hds-pagination-nav__control {
   // border width is subtracted from padding:
   padding: 0 calc(var(--token-pagination-nav-control-padding-horizontal) - 1px);
+  color: var(--token-color-foreground-primary);
```

Comment:
I committed directly to this PR to add this line. I think I may have had the color as a CSS class previously and then deleted it accidentally in a later refactor? (Not sure.)